flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/AbstractHandler.java

```diff
@@ -268,10 +266,7 @@ private CompletableFuture<Void> handleException(
                     HttpResponseStatus.SERVICE_UNAVAILABLE,
                     responseHeaders);
         } else {
-            if (throwable instanceof UnavailableDispatcherOperationException
-                    || throwable instanceof FileNotFoundException) {
-                log.debug("Job is not initialized or is finished: {}", throwable.getMessage());
-            } else {
+            if (log.isDebugEnabled()) {
```

Comment:
I wonder why we should switch to debug for a generic class which is not jdbc driver only related?

Reply:
Thanks @snuyanzin , good point! I agree with you that we should deal the exception in `ClientCoordinationsHandler` which is used to return collected results instead of the generic `AbstractHandler`

Reply:
@snuyanzin I have fixed this issue and please help to review again, thanks

Reply:
Thanks for addressing comment
I guess I'm not the best person to review here since I 'm not aware of this Flink area enough...
May be another pair of eyes would be good